HW_4/task_5.py

```diff
@@ -0,0 +1,5 @@
+import re
+with open("text", "r") as f:
+    content = f.read()
+    print(content)
+    print(re.split("\. | \.\.\.\. | \.\.\.\.\.\.", content))
```

Comment:
controls the spacecraft.A commission headed by Lew Allen
![image](https://user-images.githubusercontent.com/25909810/221375813-456af362-75fc-40bd-acce-0d1514a951c1.png)
This part of text is not separated into two sentences